src/tag.rs

```diff
@@ -1472,8 +1472,15 @@ impl<'a> Tag {
     }
 
     /// Reads AIFF file and returns ID3 Tag from it
-    pub fn read_from_aiff(path: impl AsRef<Path>) -> crate::Result<Tag> {
-        aiff::load_aiff_id3(path)
+    pub fn read_from_aiff_file(path: impl AsRef<Path>) -> crate::Result<Tag> {
```

Comment:
Could you please revert the update to the function signature? I have already released it in 0.6.0 and would prefer to not break compatibility. I realize that this is to make it follow the convention of read_from and read_from_path, but I'm afraid it's too late for that.

Reply:
Added in new commit.